cmake/DAGMC_macros.cmake

```diff
@@ -89,7 +89,7 @@ macro (dagmc_setup_options)
   endif()
 
 if (DOUBLE_DOWN)
-  find_package(DOUBLE_DOWN REQUIRED)
+  find_package(DOUBLE_DOWN 1.1.0 REQUIRED)
```

Comment:
```suggestion
  find_package(DOUBLE_DOWN REQUIRED)
   if (${DOUBLE_DOWN_VERSION} VERSION_LESS 1.1.0)
    message(FATAL_ERROR "Discovered Double Down Version: ${DOUBLE_DOWN_VERSION}. \
    Please update Double Down to version 1.1.0 or greater.")
  endif()
```

Reply:
@pshriwise is there a reason to not just let CMake take care of the error?

Reply:
Pinging @pshriwise again on this question - it may be the last thing holding up this PR